### DISCUSSION TITLE
formilyjs v2 + antd v3

### DISCUSSION THREAD

**Discussion by boborua** [alibaba/formily#2844] [Q&A 2.x]
v2版本的antd包完全是锚定antd v4来做的，有没有可能使用v3的antd呢？
目前问题：
1. antd v4 的 style/default.less 多了如@height-base等变量，直接引入webapck会报错
2. FormLayout组件中，用到了一些context，v3会报错

**Comment by RushKoala** [comment]
同样的诉求，等待～

**Reply by janryWang** [reply]
自己封装个v3版本吧
